languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/multitoken-ignore.txt

```diff
@@ -137764,4 +137764,17 @@ Kurort Seiffen
 La Chaux
 Bad Berneck
 #Obere Kyll
-Groß Kreutz
\ No newline at end of file
+Groß Kreutz
+oh, là, là
+so lala
+Regina del Lago
+Kiepenheuer & Witsch
+Warren Brown
```

Comment:
Je nachdem welche Kombinationen aus Vorname + Nachname du noch findest, könnten wir diese auch in diesen PR aufnehmen
https://github.com/languagetool-org/languagetool/pull/9734